Adafruit_Video_Looper/video_looper.py

```diff
@@ -126,8 +128,50 @@ def _is_number(iself, s):
             return True
         except ValueError:
             return False
-    
+
     def _build_playlist(self):
+        """Try to build a playlist (object) from a playlist (file).
+        Falls back to an auto-generated playlist with all files.
+        """
+        if self._config.has_option('playlist', 'path'):
```

Comment:
I think if playlist.path is not specified or empty in config, the directory.path (if in directory or copy mode) or the usb drive (if in usb_drive mode) should be scanned for m3u files. (only if playlist.format is set to m3u)
But I still agree that a seperate playlist path variable can be helpfull and should be kept.
if playlist.path is not a file but a directory I agree that it should be scanned for playlist files.